src/app/texts/chs/strings

```diff
@@ -5,7 +5,7 @@ index_title = "Help documentation"
 
 [ui]
 greetings_text = "Welcome to RR Manager!"
-tab_general = "General"
+tab_general = "设置"
```

Comment:
can you also add tab_configuration = "Configuration" in the translations in your PR? Thanks